homeassistant/components/ness_alarm/__init__.py

```diff
@@ -19,12 +20,14 @@
 
 CONF_DEVICE_HOST = 'host'
 CONF_DEVICE_PORT = 'port'
+CONF_UPDATE_INTERVAL = 'update_interval'
```

Comment:
This constant is available in `homeassistant.const`, please import that one.

Reply:
That one is soon to be deprecated 

https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/const.py#L151-L152

Is there something else to use instead?

Reply:
Nvm, answered my own question - `CONF_SCAN_INTERVAL` can be used instead:

https://github.com/home-assistant/home-assistant/pull/20924